.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Questions
+about: Have problem setting up gotify server? Feel free to ask here
+title: ''
+labels: question
+assignees: ''
+
+---
+
+<!-- 
+Alternative ways to get help:
+Official documentation - https://gotify.net/
+Community chat - https://matrix.to/#/#gotify:matrix.org
+-->
+
+**Have you read the documentation?**
+- [ ] Yes, but it does not include related information regarding my question.
+- [ ] Yes, but the steps described in the documentation do not work on my machine.
+- [ ] Yes, but I am having difficulty understanding it and wants clarification.
+
+**You are setting up gotify in**
```

Comment:
I'm unsure if we need to include this, not all questions must be about setting up gotify/server. How about just adding a commented out example question?
```
<!-- EXAMPLE
I'm having difficulties setting up my apache reverse proxy
....
my config is ...
-->
```

Reply:
Maybe we even should rename 
> Having difficulties setting up gotify server? Feel free to ask here

to
> Having difficulties with gotify/server? Feel free to ask here